python/metatomic_torch/metatomic/torch/ase_calculator.py

```diff
@@ -424,6 +430,20 @@ def calculate(
             assert len(energy.block().gradients_list()) == 0
             assert energy.block().values.shape == (1, 1)
 
+        with record_function("ASECalculator::warn_uncertainty"):
+            if calculate_energy and self._model_has_energy_uncertainty:
+                uncertainty = outputs["energy_uncertainty"].block().values
+                assert uncertainty.shape == (1, 1)
+                uncertainty = uncertainty.detach().cpu().numpy()[0, 0]
+                if uncertainty / len(atoms) > 0.1:
+                    # 100 meV/atom, which is more than 2 kcal/mol, means that this
+                    # prediction is not chemically accurate
+                    warnings.warn(
+                        "the model's energy uncertainty is larger than 100 meV/atom; "
+                        "this prediction is not chemically accurate",
```

Comment:
```suggestion
                        "the model's energy uncertainty is larger than 100 meV/atom ({uncertainty / len(atoms):.4} meV/atom), "
                        "you might be using it on a structure it can not handle",
```

Also, should the threshold be configurable or not?

Reply:
From @ceriottm 

> Should we make this threshold an option, and say that setting it to zero disables the check?
>
> Also, IDK how hard it would be to get per-atom energies and uncertainties, but I think in many cases it would be a better check than looking at the mean uncertainty.